node_server/src/main/routes/logout.ts

```diff
@@ -12,11 +12,14 @@ export default function createLogoutRouter(): Router {
    * Logs out the user by destroying the session object.
    */
   router.use('/', (req, res, next) => {
+    res.setHeader('Access-Control-Allow-Credentials', 'true');
     if (req.session) {
       req.session.destroy(err => {
         if (err) {
           next(err);
         } else {
+          res.clearCookie('connect.sid');
+          res.clearCookie('userId');
```

Comment:
Is there a way to clear all cookies? Or do we not want to do that here?

Reply:
Hmm, not an obvious way to clear all client side cookies from server (see https://expressjs.com/en/api.html). Might be able to do it from the client side.

Reply:
This seems to have been fixed for now.